chapters/bizleg-case-studies/startups/yellr.md

```diff
@@ -0,0 +1,208 @@
+    - hereME!, Free, 40 downloads
+
+## Situational Analysis
+ Yellr Specific Strengths, Weaknesses, Opportunities, & Threats 
+
+*Strengths*
+    - Free to Download
+    - Anonymity
+    - Open Source Platform
+    - Geo-fencing
+    - Entirely new and unique product
+
+*Weakness*
+    - Still in beta
+    - Not on iOS App Store (at the moment)
+    - Smartphone exclusive
+
+*Opportunities*
+    - Promote citizen engagement and journalism
+    - Expand to multiple geo-locations for varied results
+    - Potential to become a common tool for news organizations
+    - Multiple forms of monetization can be applied
+
+*Threat*
+    - Not enough demand for this app at the current moment
+    - Entirely new and unique product, people may have trouble understanding its purpose.
+
+## Market situation
+
+*Target market analysis*
+
+Market Segment:
+News Organizations and Citizens
+Demographics and Psychographics: Male or Female, “Generation X” (age 27-35), middle class, “at-homers” who like to watch the news but feel as if their own opinion isn’t being heard. They like the convenience of using their smartphone to interact with others. 
+
+*Motivation for purchase:*
+
+News Organizations: Yellr is committed to providing news organizations (in forms such as television, magazines, radio) statistical data that you wouldn’t find as an anonymous user. Organizations may use this data to create a new story or even as a source for a different topic. 
+
+*Citizens*: For the everyday citizen, this is an entirely new way to not only receive news, but to also give your input on the topic as well. 
+
+Expansion of the Target Market: In order to expand the target market, we need an aggressive social media/integrated marketing campaign that will attract a younger demographic; presumably college kids. We can start on RIT’s campus by sponsoring some events (Content Marketing), while giving them some sort of free incentive to download our app.
+
+## Competitive Environment
+
+Large Digital News Outlets
+    - iReport by CNN
+
+Small Digital News Outlets
+    - Meporter
+    - hereME!
+
+These websites/apps are similar to Yellr however there is one certain factor that makes Yellr different. All of these apps require you to create an account thus removing some factors of privacy the user has when anonymous. We’ve learned that people want to openly voice their opinion but ironically a good number of them would like their identity to remain a secret. This benefits the users security as well because there’s no possible way to be tracked down. Another advantage we have is that Yellr is completely open source meaning we can continually create new features that are asked by the community. iReport is certainly our toughest competitor however it acts as more of a website and is available to the CNN app in limited features. Also credibility has gone down in recent years due to fake stories and a decline in quality produced content. 
```

Comment:
Might some users want accounts, to eventually gain trust/notoriety? Do users expect there to be some sort of account system?